inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -0,0 +1,86 @@
+"""Visualization processing of WindTunnel scenario.
+
+This class implements various visualization capabilities for
+the WindTunnel scenario. Namely:
+    - Pressure over object; 
+    - Cutting plane;
+    - StreamLines.
+
+Currently, we only support the OpenFOAM simulator.
+"""
+from functools import singledispatchmethod
+
+import os
+import pyvista as pv
+
+from inductiva.simulation import Simulator
+from inductiva.fluids.simulators import OpenFOAM
+from inductiva.types import Path
+from inductiva.utils.visualization import Object
+
+
+class WindTunnelSimulationOutput:
+    """Post process WindTunnel simulation outputs."""
+
+    def __init__(self,
+                 sim_output_path: Path,
+                 time_step: int,
+                 simulator: Simulator = OpenFOAM()):
```

Comment:
It seems you are trying to make things overly general now, and you are getting stuck in very complex class imports and method overloadings...